src/main/java/goodgenerator/blocks/tileEntity/GTMetaTileEntity/YOTTAHatch.java

```diff
@@ -189,7 +190,7 @@ public IItemList<IAEFluidStack> getAvailableItems(IItemList<IAEFluidStack> out)
         if (host.mStorageCurrent.compareTo(BigInteger.valueOf(Long.MAX_VALUE)) > 0) {
             ready = Long.MAX_VALUE;
         } else ready = host.mStorageCurrent.longValue();
-        out.add(FluidUtil.createAEFluidStack(FluidRegistry.getFluid(host.mFluidName), ready));
+        out.add(Util.FluidUtil.createAEFluidStack(FluidRegistry.getFluid(host.mFluidName), ready));
```

Comment:
Wouldn't this create a hard dep on ae2fc, as the only optional is dependent on ae2? This led to the crash previously.
In fact it makes more sense to optional on ae2fc instead of ae2, since this is a fluid tank and while ae2 supports fluids natively, it does not interact with fluids without addons.
Another option is to simply copy the util method into the body and remove ae2fc dep entirely, since its a one-off

I don't know enough about GoodGenerator to review it well but from an initial glance that it seems strange to add a dep for a single util function, not to mention the dependency on ae2 instead of ae2fc.